### PR TITLE
Translation of the most recent changes / additions to the Chinese version

### PR DESCRIPTION
**Pull request by skyter**

### CODE REVIEW

File: EDEngineer/Resources/Data/localization.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
       "nl": "Lichtgewicht",
       "cn": null
     },
-    "Heavy Duty Hull Reinforcement": {
```
Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json:77]: could you add this back please?

**Comment by msarilar** [comment]
looks great! minor correction to apply before I can merge

File: EDEngineer/Resources/Data/localization.json (hunk 1 above)
Comment (skyter) [reply EDEngineer/Resources/Data/localization.json:77]: OK

**Review by msarilar — APPROVED** [review]
thanks a lot

**Comment by msarilar** [comment]
@skyter let me know once you're happy with the translation and I'll deploy - I mean if you have stuff in progress I can wait 👍

**Pull request merged**